src/LookupTable/components/ImportCsvModal.tsx

```diff
@@ -50,18 +49,16 @@ const ImportCsvModal: React.FC<any> = ({
     let formData: any = new FormData()
     formData.append('file', file)
 
-    const pluralTableName = pluralize.isPlural(tableName) ? tableName : pluralize.plural(tableName)
-
-    if (!tableStructureID) formData.append('tableName', pluralTableName)
+    if (!tableStructureID) formData.append('tableName', tableName)
 
     const JWT = localStorage.getItem(config.localStorageJWTKey || '')
     const authHeader = JWT ? { Authorization: 'Bearer ' + JWT } : undefined
 
     await axios
       .post(
-        config.serverREST +
-          '/admin/lookup-table/import' +
-          (tableStructureID ? '/' + String(tableStructureID) : ''),
+        `${config.serverREST}/admin/lookup-table/import${
+          tableStructureID ? '/' + String(tableStructureID) : ''
+        }?tableName=${tableName}`,
```

Comment:
As mentioned in back-end PR, I'm passing the `tableName` as a query parameter since I couldn't extract it from `formData` with the changes to the file handling.